templates/CRM/Core/Form/Field.tpl

```diff
@@ -35,6 +35,6 @@
   </td>
   <td>{if $form.$fieldName.html}{if $fieldSpec.formatter === 'crmMoney'}{$form.$fieldName.html|crmMoney}{else}{$form.$fieldName.html}{/if}{else}{$fieldSpec.place_holder}{/if}<br />
     {if $fieldSpec.description}<span class="description">{$fieldSpec.description}</span>{/if}
-    {if $fieldSpec.documentation_link}{docURL page=$fieldSpec.documentation_link.page}{/if}
+    {if $fieldSpec.documentation_lCink}{docURL page=$fieldSpec.documentation_link.page resource=$fieldSpec.documentation_link.resource}{/if}
```

Comment:
@eileenmcnaughton typo

Reply:
argh it must have recorded a key stroke in the wrong window